docs/user_guide/user_guide_overview.rst

```diff
@@ -54,6 +54,38 @@ Raster data
 This option will be implemented at a later stage.
 
 
+Aggregation Zones
+-----------------------
+Sometimes it's desirable to aggregate the exposure data into zones such as land-use, different residential zones and so forth. **FIAT** has a tool to include these aggregation zones during the creation of the exposure.csv file. To do so the aggregation zone should be in form of a vector file (e.g. *.shp* or *.gpkg*). If you want to create several aggregation zones you must provide one vector file for each zone. 
+To associate the original exposure data with the aggregation zones, utilize the **"join_exposure_aggregation_areas"** function provided by **FIAT** . This function seamlessly links each geometry in the original exposure data to its corresponding spatial aggregation zone.  
+To prepare your data create a yaml-file (*.yml*) with the following information (case-sensitive)
+SHOULD THIS BE PART OF THE CONFIGURATION OR DATA_CATALOG.YAML?   
+
+Input yaml file: 
+   - **aggregation_area_fn**: Path to the aggregation file
+   - **attribute_names**: Name of the zone attribute in your file (case-sensitive)
+   - **label_names**: Desired aggregation label for newly created aggregation zone
+
+In case you want to add several aggregation zones, you can provide several aggregation files in a comma-separated list. Make sure that the input for each variable (file path, attribute name, label name) follows the same order to assure that attribute and label names are assigned to the correct aggregation file:: 
+
+   [Example two aggregation zone files]
+   aggregation_area_fn:  C:\Users\...base_zone_aggregation.shp, C:\Users\...land_use_aggregation.shp 
+   attribute_names: ZONE_BASE, LAND_USE
+   label_names: Base Zone, Land Use
+   
+Save the yaml file and set up the **FIAT** model. As output you will receive a *.csv-file with your original exposure data together with the newly created aggregation zone(s). 
+
+It can happen that in the aggregation files polygons overlap. In this case **FIAT** will merge the information for the affected Object ID and assign both aggregation zones to the object.::
```

Comment:
Did you add "Aggregation Label: " in front of the aggregation label name (e.g., "Aggregation Label: Land Use")? Could you then also add that below in the table and explain in the text how the name is added? The FIAT Toolbox recognizes the columns starting with "Aggregation Label: " and that is how it knows which columns to use.

Reply:
Not sure what you mean exactly. 
The function adds the "Aggregation Label:"-term before the Label name, no?  
_exposure_gdf.rename(columns={attribute_name: f"Aggregation Label: {label_name}"}, inplace=True)_
Maybe we can look over this later to clarify